components/ui/Footer.js

```diff
@@ -24,8 +24,9 @@ export const Footer = () => {
       </div> */}
       {/* Docs for the Link: https://nextjs.org/docs/pages/api-reference/components/link */}
 
-      {/* TASK - React 1 week 1 */}
-      {/* Add a new list item for LINKEDIN */}
+      <li>
```

Comment:
This <li> element is outside the <ul> element. Consider moving it inside the ul after the last item to make it look  cleaner!